riscv-target/opentitan/device/rv32imc/wrap.c

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <string.h>
+
+#include "common.h"
+#include "uart.h"
+
+#define SIM_TERM_ADDR 0x10008000
+
+void dump_signature(void) {
+  extern uint32_t begin_signature[];
+  extern uint32_t end_signature[];
```

Comment:
It's strange to declare externs inside a function like this, can you move them out of the function scope.

Reply:
fixed